pkgs/development/php-packages/composer/default.nix

```diff
@@ -86,7 +97,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
     outputHashMode = "recursive";
     outputHashAlgo = "sha256";
-    outputHash = "sha256-FfFwx5E2LVDSqo2P31fqtvk2P30XnTm+TUqhNSHTt/M=";
+    outputHash = "sha256-RuHBpfU/QDIsVG5WvqfZqEiXStdtATyjyKmModR4PJQ=";
```

Comment:
When building this locally I get:
```
error: hash mismatch in fixed-output derivation '/nix/store/v80nb8gymjpn5k9li4ssza4b3qihnkmr-composer-vendor-2.8.4.drv':
         specified: sha256-RuHBpfU/QDIsVG5WvqfZqEiXStdtATyjyKmModR4PJQ=
            got:    sha256-McyO3Z4PSyC6LiWt8rsXziAIbEqOhiaT77gUdzZ6tzw=
```

Reply:
You were right, I missed that one somehow.